cloudwatch-alarms-cloudfront.tf

```diff
@@ -0,0 +1,87 @@
+variable "default_threshold" {
+  description = "The default threshold for the metric."
+  default     = 5
+}
+
+variable "default_evaluation_periods" {
+  description = "The default amount of evaluation periods."
+  default     = 2
+}
+
+variable "default_period" {
+  description = "The default evaluation period."
+  default     = 60
+}
+
+variable "default_comparison_operator" {
+  description = "The default comparison operator."
+  default     = "GreaterThanOrEqualToThreshold"
+}
+
+variable "default_statistic" {
+  description = "The default statistic."
+  default     = "Average"
+}
+
+resource "aws_cloudwatch_metric_alarm" "alarm" {
```

Comment:
Change the resource name to be specific for cloud_front, something like:

alarm_cloud_front